frontend/src/pages/JobFeed/JobFeed.tsx

```diff
@@ -24,12 +26,37 @@ export default function JobFeed() {
     },
   })
   const drawerWidth = 270
+
+  //sorts data ascending or descending by createdDate 
+  function handleChange(){
+    if (selectedSort === "asc"){
+      setselectedSort("des")
```

Comment:
If you make this an enum, not a string, TypeScript can help you make sure you use it right. You can even ensure you've covered every case.

Reply:
Also we should think about when we have multiple sorts how we want to represent that. Presumably enum for which sort and an enum for asc dec?